src/Components/App/App.js

```diff
@@ -34,17 +60,35 @@ class App extends Component {
   }
 
   render() {
+
     return (
       <div className="App">
         <Route 
           exact path='/'>
-          <Homepage />
+          <Homepage 
+            getActivityData={this.getActivityData}
+            allMovies={this.state.movies}
+          />
+        </Route>
+
+        <Route
+          exact path='/:activity'
```

Comment:
I foresee a possible issue with the dynamic path here and the path for each activity component that I included in my PR from `activity-components-ng`.  We should take a look at how I could change the name of each path in my version so that any confusion is limited.  I guess we'll see!